### PR TITLE
[ADD] setup repositories script

### PR DESCRIPTION
**Pull request by robinkeunen**
Coop IT Easy - setup repositories

Sets up the directory tree for Coop IT Easy development as set in the configuration file.
The script pulls the list of repository ub the relevant directories.
The first remote of the list will be set as origins, the others are set as remotes.
<git-user> is replaced by the user set in the command line.

### CODE REVIEW

File: cie-scripts/cie-setup-repos.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+_logger = logging.getLogger(__name__)
+logging.getLogger("sh.command").setLevel(logging.WARNING)
+
+
+def get_config(config_file):
+    try:
+        with open(config_file, 'r') as f:
+            conf = yaml.load(f)
+    except FileNotFoundError:
+        _logger.error(f'Could not find {config_file} configuration file.')
```
Comment (remytms) [inline cie-scripts/cie-setup-repos.py]: I really love f-string formatting. But the version of python on the server is python 3.5 and f-string formatting comes only with 3.6. So, for now, it should'nt be used until our servers turns to Ubuntu 18.04.

File: cie-scripts/utils.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import os
+import logging.config
+
+import yaml
+
+
+def setup_logging(
+    default_path='./documentation/cie-scripts/logging.yml',
+    default_level=logging.INFO,
+    env_key='LOG_CFG'
```
Comment (remytms) [inline cie-scripts/utils.py]: Very good usage of environment variable ! :D 
Shouldn't we use a more accurate name ? Like `<program-name>_LOG_CFG` ? `LOG_CFG` may conflict with another environment var in the future.

File: cie-scripts/utils.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+):
+    """Setup logging configuration
+
+    """
+    path = default_path
+    value = os.getenv(env_key, None)
+    if value:
+        path = value
```
Comment (remytms) [inline cie-scripts/utils.py]: What do you think about `path = os.getenv(env_key, default_path)` ?

File: cie-scripts/cie-setup-repos.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
+"""
+Coop IT Easy - setup repositories
+
+Sets up the directory tree for Coop IT Easy development as set in the configuration file.
+The script pulls the list of repository ub the relevant directories.
```
Comment (remytms) [inline cie-scripts/cie-setup-repos.py]: `ub` ?

**Comment by remytms** [comment]
@robinkeunen, Thanks for this script ! :D

**Comment by remytms** [comment]
@robinkeunen other small changes : 
`[ADD] setup repositories script]` -> `[ADD] setup repositories script`

Shouldn't the next two commits be named with `[ADD]` ?

File: cie-scripts/cie-setup-repos.py (hunk 1 above)
Comment (robinkeunen) [reply cie-scripts/cie-setup-repos.py]: Oh no, I thought the future had arrived. :-(

File: cie-scripts/utils.py (hunk 2 above)
Comment (robinkeunen) [reply cie-scripts/utils.py]: Thank you but ... I just copy pasted this snippet :p

Therefore, I think it is a default setup variable. But up to you if you want to change that variable, you're better at that :-)

File: cie-scripts/cie-setup-repos.py (hunk 4 above)
Comment (robinkeunen) [reply cie-scripts/cie-setup-repos.py]: bo, je sais pas ce qu'il s'est passé :p

File: cie-scripts/utils.py (hunk 3 above)
Comment (robinkeunen) [reply cie-scripts/utils.py]: yup, better. copy pasted.

**Pull request merged**